akka-actor/src/main/scala/akka/pattern/AskSupport.scala

```diff
@@ -261,18 +261,6 @@ trait ExplicitAskSupport {
 
 object AskableActorRef {
 
-  /**
```

Comment:
do we not still need to be binary compatible?

Reply:
These were once added to make 2.5 binary compatible to 2.4. Because 2.4 is now EOL, I don't think we need to be binary compatible with it anymore, and 2.5 code could never reach those so removing this does not make use binary incompatible with 2.5 (I think I added a comment to the mima exclusion)